packages/image/src/load.ts

```diff
@@ -1,15 +1,10 @@
-export interface ImageSource {
-  defaultSrc: string;
-  webpSrc?: string;
-}
-
 /**
  * Loads the given images. If WebP is WebP source is provided, it will load that. Otherwise, it loads the default source.
```

Comment:
```suggestion
 * Loads an image from the given source URL and returns a Promise that resolves to the loaded image.
```

There is no longer webP 😁